addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -116,7 +116,31 @@
         </t>
     </template>
 
+    <template id="task_link_preview_front_end" inherit_id="portal.frontend_layout" primary="True">
+        <xpath expr="//meta[@property='og:image']" position="replace">
+            <t t-if="preview_object.displayed_image_id">
+                <meta property="og:image" t-attf-content="/web/image/#{preview_object.displayed_image_id.id}/300x200?access_token=#{preview_object.displayed_image_id.generate_access_token()[0]}"/>
+            </t>
+            <meta property="og:image" t-attf-content="/web/binary/company_logo?company={{ preview_object.company_id.id }}"/>
```

Comment:
Note that in email layouts, we use 'uses_default_logo' (computed field to know if company has a logo or is using the default one) to avoid having references to logo when it is the default (not nice) one. Could be of use here :) .